### PR TITLE
html_lab05

### PR DESCRIPTION
**Pull request by violette-git**

### CODE REVIEW

**Comment by violette-git** [comment]
I really tried to get the hang of the rows and cols with this one.

File: code/grant/html_lab05/static/styles.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
+
+
+
+
+
+
```
Comment (cdmichaelb) [inline code/grant/html_lab05/static/styles.css:6]: Moving forward you should be able to completely move away from the use of raw CSS. You still want to remember it for when you need it, but you shouldn't need it super often.

File: code/grant/html_lab05/templates/index.html [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.15.1/devicon.min.css">
+
+    <link rel="stylesheet" href="{{url_for('static', filename='styles.css')}}">
+    <title>Borg's Burritos</title>
+</head>
+<!-- HEAD SECTION END -->
+
+<body>
+    <section class="container-fluid bg-dark text-light" id="main">
```
Comment (cdmichaelb) [inline code/grant/html_lab05/templates/index.html:21]: Nice box usage and responsiveness. Looks like you've really got this down now!

**Review by cdmichaelb — APPROVED** [review]
Great use of bootstrap to create responsive pages with a mobile-first design. The only suggestion I can make is to use more semantic elements. Often times it's just a matter of naming something a nav or section instead of a div. https://www.w3schools.com/html/html5_semantic_elements.asp

**Pull request merged**